Formula/ncmpcpp.rb

```diff
@@ -31,6 +31,7 @@ class Ncmpcpp < Formula
   depends_on "pkg-config" => :build
   depends_on "libmpdclient"
   depends_on "readline"
+  depends_on "ncurses"
```

Comment:
The question is specifically whether they intend to require this in the next release, which would be quite unfortunate.

Also, since this is only required for head, it should be moved into the `head do` block.